src/commands/client/login.js

```diff
@@ -37,7 +37,10 @@ module.exports = async args => {
   if (!res.data || !res.data.text) {
     return error(`Failed to login using creds: ${token}`);
   }
-  const cmd = shell.exec('nim auth login --auth ' + res.data.text);
+  const { apiHost, auth } = JSON.parse(res.data.text);
```

Comment:
I think this PR is generally correct and in the right direction modulo this change. To get the api host you had to already have invoked `cli_login` on the appropriate api host to begin with. This is circular if you're trying to login to a client for the first time and there is no set credential.